docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md

```diff
@@ -3,60 +3,51 @@ title: Using Visual Studio Tools for Docker (Visual Studio on Windows)
 
 ## Configuring your local environment
 
 With the latest versions of Docker for Windows, it is easier than ever to develop Docker applications because the setup is straightforward, as explained in the following references.
 
 **More info:** To learn more about installing Docker for Windows, go to <https://docs.docker.com/docker-for-windows/>.
 
-If you're using Visual Studio 2015, you must have Update 3 or a later version plus the Visual Studio Tools for Docker.
-
-**More info:** For instructions on installing Visual Studio, go to [https://visualstudio.microsoft.com/\
-products/vs-2015-product-editions](https://visualstudio.microsoft.com/products/vs-2015-product-editions).
+**More info:** For instructions on installing Visual Studio, go to [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/).
 
 To see more about installing Visual Studio Tools for Docker, go to <http://aka.ms/vstoolsfordocker> and <https://docs.microsoft.com/aspnet/core/host-and-deploy/docker/visual-studio-tools-for-docker>.
 
-If you're using Visual Studio 2017, Docker support is already included.
-
-## Using Docker Tools in Visual Studio 2015
-
-The Visual Studio Tools for Docker provides a consistent way to develop and validate locally your Docker containers for Linux in a Linux Docker host or VM, or your Windows Containers directly on Windows.
-
-If you're using a single container, the first thing you need to begin is to turn on Docker support into your .NET Core project. To do this, right-click your project file, as shown in Figure 4-25.
-
-![https://i1.visualstudiogallery.msdn.s-msft.com/0f5b2caa-ea00-41c8-b8a2-058c7da0b3e4/image/file/205468/1/add-docker-support.png](./media/image31.png)
-
-Figure 4-25: Turning on Docker support for your Visual Studio project
+Docker support is included in Visual Studio 2017.
 
 ## Using Docker Tools in Visual Studio 2017
 
-When you add Docker support to a service project in your solution (see Figure 4-26), Visual Studio is not just adding a DockerFile file to your project, it also is adding a service section in your solution's docker-compose.yml files (or creating the files if they didn't exist). It's an easy way to begin composing your multicontainer solution; you then can open the docker-compose.yml files and update them with additional features.
+When you add Docker support to a service project in your solution (see Figure 4-26), Visual Studio adds a *Dockerfile* to your project without adding support for multiple containers. This configuration is ideal for single-project solutions with a single container.
```

Comment:
~~When you add Docker support to a service project in your solution (see Figure 4-26), Visual Studio adds a *Dockerfile* to your project without adding support for multiple containers.~~
When adding Docker support to a project (see Figure 4-26), Visual Studio adds a *Dockerfile* to the project root. Container orchestration support, via Docker Compose, is added by default in Visual Studio 2017 versions 15.7 or earlier. Container orchestration support is an opt-in feature in Visual Studio 2017 versions 15.8 or later, in which case Docker Compose and Service Fabric are supported.